src/Core/GeometryPrimitives/BBoxBase.h

```diff
@@ -38,9 +38,9 @@ namespace Geometry {
 class SCISHARE BBoxBase
 {
 protected:
-  Point cmin_;
-  Point cmax_;
-  bool is_valid_;
+ bool is_valid_;
```

Comment:
This is the wrong way to fix that warning--it might break IO. Change the constructor list to match the declared order.

Reply:
What's the difference? How can this break IO?

Reply:
These types are read in sequential byte order. If the layout order changes, then the data will be read in the wrong order.

Reply:
That makes sense. I already changed this by the way

Reply:
Cool thanks